docs/site/example_rpc.md

```diff
@@ -0,0 +1,29 @@
+---
+lang: en
+title: RPC Example
+keywords: LoopBack 4.0
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/examples_rpc.html
```

Comment:
We are keeping all LoopBack4 documentation files inside loopback-next. Please move this file to https://github.com/strongloop/loopback-next/tree/master/docs/site.

See also https://loopback.io/doc/en/contrib/doc-contrib.html#how-loopback-documentation-works